src/community/geofence-server/pom.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- set parent pom to community pom -->
+    <parent>
+        <groupId>org.geoserver</groupId>
+        <artifactId>community</artifactId>
+        <version>2.8-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.geoserver.community</groupId>
+    <artifactId>gs-geofence-server</artifactId>
+    <packaging>jar</packaging>
+    <version>2.8-SNAPSHOT</version>
+    <name>GeoFence security internal server</name>
```

Comment:
Usually name is shorter :) and you can add a description if needed.
Do you want to list yourself as a developer/contact point?